### PR TITLE
Adding a "permissions" section so it stands out to new users

### PR DESCRIPTION
**Pull request by ykhli**
This is a minor change that makes the "permission" section stands out better for users.

### CODE REVIEW

File: content/cloud-docs/workspaces/variables/managing-variables.mdx [hunk 1]
```diff
@@ -11,10 +11,12 @@ You can set variables specifically for each workspace or you can create variable
 
 ~> **Note:** Variable sets are in beta.
 
+### Permissions
```
Comment (laurapacilio) [inline content/cloud-docs/workspaces/variables/managing-variables.mdx:14]:
```suggestion
## Permissions
```

File: content/cloud-docs/workspaces/variables/managing-variables.mdx [hunk 2]
```diff
@@ -11,10 +11,12 @@ You can set variables specifically for each workspace or you can create variable
...
 You must have [`read variables` permission](/cloud-docs/users-teams-organizations/permissions#general-workspace-permissions) to view the variables for a particular workspace and to view the variable sets in your organization.
 
 To create new variable sets and apply them to workspaces, you must be part of a team with [manage workspaces](/cloud-docs/users-teams-organizations/permissions#organization-owners) permissions. To create and edit workspace-specific variables within a workspace, you must have [read and write variables](/cloud-docs/users-teams-organizations/permissions#general-workspace-permissions) for that workspace.
 
+### Create and Edit Variables
```
Comment (laurapacilio) [inline content/cloud-docs/workspaces/variables/managing-variables.mdx:19]: Let's leave this as part of the introduction (it's just meant to introduce the options for managing variables). And lets move it to be above the variables section, after the intro that's already there. And I think we can just say "You can create and edit variable sets through...." and remove the permissions statement as it's now redundant and we have a beautiful new flagged section.

**Review by laurapacilio — APPROVED** [review]
A couple of suggestions and then pre-approving so I don't block you. Thank you for doing this!!

File: content/cloud-docs/workspaces/variables/managing-variables.mdx [hunk 3]
```diff
@@ -9,17 +9,20 @@ description: >-
 
 You can set variables specifically for each workspace or you can create variable sets to reuse the same variables across multiple workspaces. Refer to the [variables overview](/cloud-docs/workspaces/variables) documentation for more information about variable types, scope, and precedence. You can also set variable values specifically for each run on the command line.
 
+You can create and edit workspace-specific variables through:
```
Comment (ykhli) [inline content/cloud-docs/workspaces/variables/managing-variables.mdx:12]: @laurapacilio I moved the create/edit workspace specific variables section here. Making sure this is actually the right place!

Comment (laurapacilio) [reply content/cloud-docs/workspaces/variables/managing-variables.mdx:12]: yes, thank you!

**Pull request merged**